.github/workflows/release-drafter.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - main
-  pull_request:
+  pull_request_target:
```

Comment:
This also comes with some [security concerns](https://securitylab.github.com/resources/github-actions-preventing-pwn-requests/) and we should evaluate carefully prior to merging.